### PR TITLE
Integration tests for run page

### PR DESCRIPTION
**Pull request by callunity**
GLT-1969

### CODE REVIEW

File: miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/RunPage.java [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    partitions.forEach(partition -> partitionsTable.checkBoxForRow(partition));
+    partitionsSection.findElement(By.linkText(titleText)).click();
+    waitUntil(visibilityOf(dialog));
+    waitUntil(textToBe(By.className("ui-dialog-title"), titleText));
+    dialog.findElement(By.linkText(option)).click();
+  }
+
+  private void clickFirstPoolTile(boolean extraWait, String searchText) {
+    if (extraWait) {
+      waitUntil(textToBe(By.className("ui-dialog-title"), "Select Pool"), 60);
```
Comment (djcooke) [inline miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/RunPage.java]: Wow, searching by "ready to run" is pretty slow for some reason...

File: miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/element/DataTable.java [hunk 2]
```diff
@@ -77,6 +79,14 @@ public String getTextAtCell(String columnHeading, int rowNum) {
     return cell.getText();
   }
 
+  public void checkBoxForRow(int rowNum) {
+    List<WebElement> checkbox = getCell(Columns.SORT, rowNum).findElements(By.tagName("input"));
+    if (checkbox.isEmpty()) {
+      throw new IllegalArgumentException("Row " + rowNum + " does not have a checkbox to click.");
+    }
+    checkbox.get(0).click();
```
Comment (djcooke) [inline miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/element/DataTable.java]: Use `findElementIfExists` and null check instead. Includes check for unexpected multiple results

Comment (callunity) [reply miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/element/DataTable.java]: In this case, the "find" needs to be only on the results of `getCell`. `findElementIfExists` will look over an entire page, and tables with more than one row will cause it to throw an error.

File: miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/RunPage.java (hunk 1 above)
Comment (callunity) [reply miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/RunPage.java]: Yeah, I'm not sure why that is.

**Pull request merged**